### ISSUE TITLE
postgres password

### ISSUE THREAD

**Issue by CzechJiri** [Mashape/docker-kong#44]
ideally host/port/user/database/password (or at bare minimum password) should be supplied via environment variables (or whatever other way), but it should not be left out to defaults without any option to change it

**Comment by thibaultCha** [comment]
The upcoming 0.9 release will have support for overriding any property with environment variables, thus should be able to solve this use-case.

**Comment by yunspace** [comment]
@CzechJiri I've made a fork here https://github.com/LittleBayDigital/docker-kong-service which supports env vars for postgres and cassandra. I'm currently using this image at 2 different companies via a custom Rancher catalog. Feel free to try out the fork until 0.9 comes out.

@thibaultCha good to hear env var override will be available in 0.9. I'll gladly point back my Rancher catalog back to the official image once it's out

**Comment by CzechJiri** [comment]
thank you @thibaultCha really nice improvement! Would it be worth porting features @yunspace added to the fork to official docker in 0.8?

@yunspace you are reading my mind and actually talking about identical use case I was going for!

**Comment by yunspace** [comment]
Haha thanks @CzechJiri, this feature was originally asked here:   https://groups.google.com/forum/#!topic/konglayer/mfjBUwQHHHk and what I did in the fork was actually exactly what @thibaultCha suggested in that forum.

The reason why I didn't create a PR was because I felt:

1. the `sed` approach was very hacky, catering for only a handful of hardcoded environment variables and not all possible env vars.
2. the fork I made was very Rancher specific. I.e. it contains code which calls the Rancher meta service, which has nothing to do with docker-kong.

If the devs are working on a nicer solution in Kong 0.9, I think it's best to do away with the messy `sed` calls and retrofit my rancher logic on top of 0.9.
